src/renderer/src/components/song/song-item/SongItem.tsx

```diff
@@ -27,7 +30,9 @@ const SongItem: Component<SongItemProps> = ({
 }) => {
   const showSignal = createSignal(false);
   const [_coords, setCoords] = createSignal<[number, number]>([0, 0], { equals: false });
-  let item;
+  const [avgColor, setAvgColor] = createSignal<string>("black"); // Default color
```

Comment:
Please use fully qualified names for naming considering whole codebase uses this standard. Change to: `averageColor` and `setAverageColor`